content.md

```diff
@@ -1,23 +1,278 @@
+
+In the terminal run this command:
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+After the initial install you will be instructed to run 2 commands in your terminal. Please copy the specific commands from your terminal as they are specific to your system
+
+```bash
+# Example (please use the commands listed in your terminal)
+(echo; echo 'eval "$(/opt/homebrew/bin/brew shellenv)"') >> /Users/dpi-pttl-3/.zprofile
+
+eval "$(/opt/homebrew/bin/brew shellenv)"
+```
+
+## Setup a Ruby Version Manager
+
+**rbenv** is a lightweight Ruby version management tool, simplifying the process of managing multiple Ruby installations. While alternatives like **asdf** offer multi-language support and **rvm** provides additional features like gemset management, **rbenv** is recommended for its simplicity and ease of use, making it ideal for developers seeking straightforward Ruby version management.
+
+### What is rbenv
+
+`rbenv` is a tool used in Ruby development to manage different versions of the Ruby programming language on a single machine. Here's what it does:
+
+1. **Version Management**: `rbenv` allows developers to install and switch between multiple versions of Ruby on their system. This is particularly useful when working on projects that require different versions of Ruby due to compatibility or dependency reasons.
+2. **Isolation**: `rbenv` provides a way to isolate different Ruby environments for different projects. Each project can have its own Ruby version specified, ensuring that dependencies and configurations do not conflict between projects.
+3. **Integration with Shell**: `rbenv` integrates with the shell environment, allowing users to set a specific Ruby version globally or per project using simple commands. This ensures that the correct Ruby version is used when running Ruby scripts or applications.
+4. **Plugin System**: `rbenv` has a plugin system that extends its functionality. Users can install plugins to add features such as automatic switching of Ruby versions based on the directory or environment variables.
+
+Overall, `rbenv` simplifies the process of managing Ruby versions and environments, making it easier for developers to work on multiple projects with different Ruby requirements without conflicts.
+
+### Installing rbenv
+
+[rbenv github repo](https://github.com/rbenv/rbenv)
+
+In the terminal run this command:
+
+```bash
+brew install rbenv ruby-build
+```
+
+Load rbenv in your shell
+
+```bash
+# run this and follow the printed instructions:
+rbenv init
+```
+
+To automatically load rbenv please append this code to your ~/.zshrc file
+
+```bash
+# Open ~/.zshrc in VSCode by typing this into the terminal
+code ~/.zshrc
+
+#Paste this code in your file, save the file and close it
+eval "$(rbenv init - zsh)"
+```
+
+Close your terminal and launch a new one for changes to take effect
+
+### Choose a version of Ruby to install
+
+<aside>
```

Comment:
no need for aside here I think. in learn it will move it to the right (like a footnote). So either include the header in the aside "### Choose a version of Ruby to install" or remove the `<aside>` tags